manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -335,7 +293,7 @@ spec:
     - name: SCRIPT
       value: |
         if [ "$(params.upon-end)" == "stop" ]; then
-          oc scale deployment.apps/$(tasks.sanitise-model-name.results.output-string)-$(params.model-version) --replicas=0
+          oc scale deployment.apps/$(params.model-name)-$(params.model-version) --replicas=0
         elif [ "$(params.upon-end)" == "delete" ]; then
           oc delete all --selector=app=$(params.model-name)-$(params.model-version)
         elif [ "$(params.upon-end)" == "keep" ]; then
```

Comment:
`s3-secret` and `s3-ssl-cert` workspaces in this pipeline should be removed as well.